third_party/move/move-compiler-v2/src/lib.rs

```diff
@@ -168,6 +169,9 @@ pub fn bytecode_pipeline(env: &GlobalEnv) -> FunctionTargetPipeline {
     pipeline.add_processor(Box::new(LiveVarAnalysisProcessor()));
     if safety_on {
         pipeline.add_processor(Box::new(ReferenceSafetyProcessor {}));
+        pipeline.add_processor(Box::new(ExplicitDrop {}));
+        // TODO(#11415): cannot turn this on until the issue solved
```

Comment:
So lets fix this first before landing. Looking at #11415 it appears this maybe just a problem of using `.before` instead of `.after` when computing copies.

Reply:
Perhaps the example is misleading; the real cause is the `force_copy_hack`.

Reply:
You can fix the `force_copy_hack` in this PR, you know, if that is required.